source/db/protocols/en-protocols.json

```diff
@@ -72,6 +72,15 @@
     }]
   },
   {
+    "name": "OMEMO",
```

Comment:
Unfortunately localized protocol definitions don't inherit from the English version like projects do, so you'll need to make this same change to all other protocol JSON files. Sorry :(

I'd suggest writing a `for` loop in the shell with `patch` or something.